sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -326,20 +326,6 @@ for (auto keys = client.GetPropertiesOfKeys(); keys.HasPage(); keys.MoveToNextPa
     }
 ```
 
-<!-- ### Encrypt and Decrypt
-
-This example creates a `CryptographyClient` and uses it to encrypt and decrypt with a key in Azure Key Vault.
-
-```cpp
-byte[] plaintext = Encoding.UTF8.GetBytes("A single block of plaintext");
-
-// encrypt the data using the algorithm RSAOAEP
-EncryptResult encryptResult = cryptoClient.Encrypt(EncryptionAlgorithm.RsaOaep, plaintext);
-
-// decrypt the encrypted data.
-DecryptResult decryptResult = cryptoClient.Decrypt(EncryptionAlgorithm.RsaOaep, encryptResult.Ciphertext);
-``` -->
```

Comment:
Can we fix this and show the sample rather than removing it? Or is it redundant now or no longer relevant for some reason?

https://github.com/Azure/azure-sdk-for-cpp/blob/4eb33a00152d0bd3f95a4826933478151ddaed8f/sdk/keyvault/azure-security-keyvault-keys/src/cryptography/cryptography_client.cpp#L120